src/components/Home.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+import { getNewsArticles } from '../utils/api';
+// import { Link } from 'react-router-dom';
+
+export const Home = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getNewsArticles().then((articlesFromApi) => {
+      setArticles(articlesFromApi);
+    });
+  }, []);
+
+  console.log(articles);
+  return (
+    <div className='login-page'>
+      <ul className='article-list'>
+        {articles.map((data) => {
+          return (
+            <li key={data.article_id}>
```

Comment:
Great work so far John! If I were being nitpicky(which you have not left me much choice as there is nothing really to nitpick), we could make this component a tad more clean, by abstracting all the HTML for each Article, to its own component.